opm/simulators/timestepping/TimeStepControl.cpp

```diff
@@ -177,16 +177,20 @@ namespace Opm
     double PIDAndIterationCountTimeStepControl::
     computeTimeStepSize( const double dt, const int iterations, const RelativeChangeInterface& relChange,  const double simulationTimeElapsed ) const
     {
-        double dtEstimate = BaseType :: computeTimeStepSize( dt, iterations, relChange, simulationTimeElapsed);
-
-        // further reduce step size if to many iterations were used
-        if( iterations > target_iterations_ )
-        {
-            // if iterations was the same or dts were the same, do some magic
-            dtEstimate *= double( target_iterations_ ) / double(iterations);
+        double dtEstimatePID = BaseType :: computeTimeStepSize( dt, iterations, relChange, simulationTimeElapsed);
+
+        // adjust timesteps based on target iteration
+        double dtEstimateIter;
+        if (iterations > target_iterations_) {
+            double percent = double(iterations - target_iterations_) / target_iterations_;
```

Comment:
The variable `percent` is not a percentage, which is a little misleading, `off_target_fraction` or something would perhaps be better?